recipes/tests.py

```diff
@@ -1,3 +1,46 @@
 from django.test import TestCase
+from rest_framework.test import APIClient
+from rest_framework.reverse import reverse
+from rest_framework import status
 
-# Create your tests here.
+
+class CreateIngredientTestCase(TestCase):
+    url = reverse('create_ingredient')
+
+    def create_response(self, name):
+        client = APIClient()
+        response = client.post(
+            self.url,
+            {'name': name},
+            format='json'
+        )
+        return name, response
+
+    def test_create_ingredient(self):
+        name, response = self.create_response('test_ingredient')
```

Comment:
You can assign `test_ingredient` to a variable. In long tests, updating a parameter may be complicated. You should not forget to update `test_ingredient` in rest of the test.